### PR TITLE
feat:implement ledgers-bank-account-access-rest-api

### LINKED ISSUE #23: Create Bank Account Access REST API (ledgers-bank-account-access-rest-api)
Develop the REST API that will expose operations for managing bank account access.
**Steps:**
Define API endpoints for:

- Creating, updating, and deleting BankAccountAccess entries.
- Retrieving account access details.
- Ensure proper request/response format for each operation.

**Acceptance Criteria:** REST API is functional and can be used to manage BankAccountAccess via HTTP requests.

### PR DESCRIPTION
**Pull request by NkwaTambe**
### Develop the REST API that will expose operations for managing bank account access.

### CODE REVIEW

File: ledgers-bank-account-access-management/ledgers-bank-account-access-rest-api/src/main/java/de/adorsys/ledgers/baam/api/resource/BankAccountAccessAPI.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ResponseEntity<List<BankAccountAccess>> getAllAccesses();
+
+    @PutMapping("/{accessId}")
+    @Operation(summary = "Update Bank Account Access", description = "Updates an existing bank account access entry")
+    ResponseEntity<BankAccountAccess> updateAccess(@PathVariable String accessId, @RequestBody BankAccountAccess access);
+
+    @DeleteMapping("/{accessId}")
+    @Operation(summary = "Delete Bank Account Access", description = "Deletes a bank account access entry by ID")
+    ResponseEntity<Void> deleteAccess(@PathVariable String accessId);
+}
```
Comment (Koufan-De-King) [inline ledgers-bank-account-access-management/ledgers-bank-account-access-rest-api/src/main/java/de/adorsys/ledgers/baam/api/resource/BankAccountAccessAPI.java]: Hello @NkwaTambe, 
The Id field for the bank-account-access entity is ```Id```, not ```accessId```, isn't it?

**Review by Koufan-De-King — CHANGES_REQUESTED** [review]

File: ledgers-bank-account-access-management/ledgers-bank-account-access-rest-api/src/main/java/de/adorsys/ledgers/baam/api/resource/BankAccountAccessAPI.java (hunk 1 above)
Comment (NkwaTambe) [reply ledgers-bank-account-access-management/ledgers-bank-account-access-rest-api/src/main/java/de/adorsys/ledgers/baam/api/resource/BankAccountAccessAPI.java]: Hahaha, thanks @Koufan-De-King , probably errors with the naming convention, it always right to follow best rest endpoint naming practices..

please review new changes

**Review by Koufan-De-King — APPROVED** [review]

**Review by yvanhenang — APPROVED** [review]
Good job.

**Review by Arielpetit — APPROVED** [review]
Looks Good

**Review by hanswizzy — APPROVED** [review]

**Pull request merged**